src/libmongoc/tests/TestSuite.c

```diff
@@ -857,36 +863,6 @@ TestSuite_PrintJsonFooter (FILE *stream) /* IN */
    fflush (stream);
 }
 
-
-static int
```

Comment:
A distinct `TestSuite_RunSerial` is no longer needed. If there are no match patterns specified `TestSuite_RunAll` considers anything to match.